src/ROUTES.ts

```diff
@@ -486,6 +486,14 @@ const ROUTES = {
         route: 'workspace/:policyID/workflows',
         getRoute: (policyID: string) => `workspace/${policyID}/workflows` as const,
     },
+    WORKSPACE_WORKFLOWS_AUTOREPORTING_FREQUENCY: {
+        route: 'workspace/:policyID/workflows/autoreporting/frequency',
+        getRoute: (policyID: string) => `workspace/${policyID}/workflows/autoreporting/frequency` as const,
```

Comment:
Just noticed that the doc specifies the route as `workspace/:policyID/settings/workflows/auto-reporting-frequency`
```suggestion
        getRoute: (policyID: string) => `workspace/${policyID}/workflows/auto-reporting-frequency` as const,
```

Reply:
nitpick: the suggestion is still missing `settings` in route